### PR TITLE
fix public gallery strings and links

### PR DESCRIPTION
**Pull request by davidsbailey**
### Description

* makes the "View more Games and Events projects" link work
* shows better header text when viewing more artist or playlab projects
* rearranges a few existing strings

### Screencast

public-gallery-strings-and-links

### CODE REVIEW

**Comment by davidsbailey** [comment]
@caleybrock can you please review? @poorvasingal can you comment on the UI screens you see in the anigif shown in the PR description?

File: apps/src/templates/projects/ProjectCardGrid.jsx [hunk 1]
```diff
@@ -134,21 +134,33 @@ const ProjectCardGrid = React.createClass({
...
-                labKey="playlab"
-                labName={i18n.projectTypeAllProjectsPlaylab()}
-                labViewMoreString={i18n.projectsViewAll()}
-                projectList={projectLists.playlab}
-                numProjectsToShow={numProjects}
-                galleryType={this.props.galleryType}
-                navigateFunction={this.viewAllProjects}
-                isDetailView={true}
-              />
+            <ProjectAppTypeArea
```
Comment (caleybrock) [inline apps/src/templates/projects/ProjectCardGrid.jsx]: nit: some of these `ProjectAppTypeArea` components aren't indented properly.

File: apps/src/templates/projects/ProjectCardGrid.jsx [hunk 2]
```diff
@@ -134,21 +134,33 @@ const ProjectCardGrid = React.createClass({
...
+              galleryType={this.props.galleryType}
+              navigateFunction={this.viewAllProjects}
+              isDetailView={true}
+            />
             }
             {this.state.showApp === 'artist' &&
               <ProjectAppTypeArea
                 labKey="artist"
-                labName={i18n.projectTypeAllProjectsArtist()}
+                labName={showMoreProjects ? i18n.projectGroupArtistAllProjects() : i18n.projectTypeAllProjectsArtist()}
```
Comment (caleybrock) [inline apps/src/templates/projects/ProjectCardGrid.jsx:163]: is it on purpose that some of these have a conditional and some of them don't?

Comment (davidsbailey) [reply apps/src/templates/projects/ProjectCardGrid.jsx:163]: yep, only the ones that are changing as part of this overall public gallery change have conditionals, so for example App Lab and Game Lab have the same strings before and after so there is no conditional. Also, Minecraft didn't exist before as a category, so there is a conditional for whether it shows up but no conditional for its display name. The work item list describing this work is here, showing the "list of categories in order": https://docs.google.com/document/d/1NYgN4G3FDlyTevKGE4_5wNzYkB36BJKZzsLn_ESLGmA/edit#

File: apps/i18n/common/en_us.json [hunk 3]
```diff
@@ -586,13 +586,17 @@
...
   "projectGroupEvents": "Games with Events",
+  "projectGroupEventsAllProjects": "All Games with Events Projects",
   "projectGroupEventsViewMore": "View more Games with Events projects",
   "projectGroupMinecraft": "Minecraft",
+  "projectGroupMinecraftAllProjects": "All Minecraft Projects",
   "projectGroupMinecraftViewMore": "View more Minecraft projects",
   "projectGroupPlaylab": "Stories and Games with Play Lab",
-  "projectGroupPlaylabViewMore": "View more Play Lab projects",
+  "projectGroupPlaylabAllProjects": "All Stories and Games with Play Lab Projects",
+  "projectGroupPlaylabViewMore": "View more Stories and Games with Play Lab projects",
```
Comment (caleybrock) [inline apps/i18n/common/en_us.json]: These are very long and will likely be longer is other languages, can we get away with just "View more Play Lab projects" or is this string required?

Comment (davidsbailey) [reply apps/i18n/common/en_us.json]: @poorvasingal , what do you think about these strings? you can see them in context in the anigif

**Comment by poorvasingal** [comment]
I think the following set will work:

- All Play Lab Projects (View more Play Lab projects)
- All Games with Events (View more Games with events)

@caleybrock and @davidsbailey - Does that sound good to you both?

**Pull request merged**